core/web/router.go

```diff
@@ -327,7 +331,7 @@ func loggerFunc() gin.HandlerFunc {
 		buf, err := ioutil.ReadAll(c.Request.Body)
 		if err != nil {
 			logger.Error("Web request log error: ", err.Error())
-			c.Next()
+			c.AbortWithStatus(http.StatusUnprocessableEntity)
```

Comment:
I actually think this warrants a 400!

400 Bad Request
The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, **size too large**

Reply:
Just explicitly stating why I agree, even if it's obvious. There's 413 `http.StatusRequestEntityTooLarge` which is effectively `size too large`. But we can't be certain that that was the reason `ioutil.ReadAll` chose to error, so `http.StatusBadRequest` is used to represent all 4** errors, including `size too large`.